context/app/static/js/pages/Organ/Organ.tsx

```diff
@@ -12,15 +12,20 @@ import Samples from 'js/components/organ/Samples';
 import DatasetsBarChart from 'js/components/organ/OrganDatasetsChart';
 import Section from 'js/shared-styles/sections/Section';
 
+import { OrganFile } from 'js/components/organ/types';
 import { FlexRow, Content } from './style';
 
-function Organ({ organ }) {
-  const summaryId = 'Summary';
-  const hraId = 'Human Reference Atlas';
-  const referenceId = 'Reference-Based Analysis';
-  const assaysId = 'Assays';
-  const samplesId = 'Samples';
+interface OrganProps {
+  organ: OrganFile;
+}
+
+const summaryId = 'Summary';
+const hraId = 'Human Reference Atlas';
+const referenceId = 'Reference-Based Analysis';
+const assaysId = 'Assays';
+const samplesId = 'Samples';
```

Comment:
Would this make more sense as an obj?

Reply:
These were defined as instance variables inside the component previously, I pulled them out in the same form they were already in. They are referenced individually in the method body when providing the ID to each section.